frontend/src/components/pages/events/EventDetails.tsx

```diff
@@ -263,6 +270,19 @@ const EventDetails: React.FC<Props> = ({ eventId }) => {
                   onClick={handleClick}
                   currentTime={timeData.serverTime}
                 />
+                {eventData.event.products.length &&
+                  eventData.event.userAttendance?.isSignedUp &&
+                  (eventData.event.userAttendance.hasBoughtTicket ? (
```

Comment:
```suggestion
                {eventData.event.products.length &&
                  eventData.event.userAttendance?.isSignedUp &&
                  (eventData.event.userAttendance.hasBoughtTicket ? (
```

A bit unwieldy, preferably refactor into a descriptive variable